lithuanian/apsisukti/apsisukti.md

```diff
@@ -1 +1,3 @@
-Jis apsisuko ir pamatė
+Jis apsisuko ir pamatė, sviesos sinduleli tolumoje.
+Jis svarste ar [eiti](eiti/sviesa.md) i sviesa, nes buvo girdejes ne kokiu atsiliepimu apie tai,
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.